docs/gateway-configuration/network-advanced.md

```diff
@@ -0,0 +1,32 @@
+# Advanced Network Settings
+
+For devices configured to use [NetworkManager](https://networkmanager.dev), it is possible to configure multiple VLAN interfaces. Take a look at our [installer profiles](../getting-started/install-kura.md#installer-types) to know which are supported.
```

Comment:
copy-paste mistake, should be `it is possible to fine tune network traffic settings.` rather than VLANs all over again :)
Also, I didn't follow @marcellorinaldo previous tip yet
> the "Network Configuration" section specifying that the advanced tab is available only on generic profiles
> 
> > Is it specifically generic profiles or only on profiles using Kura's new network management?
>
> I would stick with just "generic profiles", as here: http://eclipse.github.io/kura/docs-release-5.4/getting-started/install-kura/

to keep the same opening as VLAN Configuration page, but can change both if needed

Reply:
> but can change both if needed

Let's follow @marcellorinaldo suggestion and change both to "generic profiles" 👍